packages/react/navigation-menu/src/NavigationMenu.tsx

```diff
@@ -28,6 +28,7 @@ type Direction = 'ltr' | 'rtl';
  * -----------------------------------------------------------------------------------------------*/
 
 const NAVIGATION_MENU_NAME = 'NavigationMenu';
+const DEFAULT_DELAY_DURATION = 700;
```

Comment:
>  If we're too slow here it will exacerbate the issue of users thinking it's a click interaction and immediately closing the menu.

Yes, that's exactly what I thought too.